From 8a4b912a5af499f6c733514dd917220bff819725 Mon Sep 17 00:00:00 2001
From: Meticulus <theonejohnnyd@gmail.com>
Date: Sat, 7 Jan 2017 09:40:46 -0600
Subject: [PATCH] bt: fix bluetooth.

-Don't ask me why this works, I have no idea but somehow blocking
opcode 0x40f fixes pairing.

Change-Id: I4baa809355c388dddec1ceeda9df3e0d1ccec305
---
 hci/src/hci_layer.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/hci/src/hci_layer.c b/hci/src/hci_layer.c
index ed1c7ee..c02dbe0 100644
--- a/hci/src/hci_layer.c
+++ b/hci/src/hci_layer.c
@@ -357,6 +357,8 @@ static void transmit_command(
 
   uint8_t *stream = command->data + command->offset;
   STREAM_TO_UINT16(wait_entry->opcode, stream);
+  if(wait_entry->opcode == 0x40f)
+	return;
   wait_entry->complete_callback = complete_callback;
   wait_entry->status_callback = status_callback;
   wait_entry->command = command;
-- 
1.9.1

